{{cookiecutter.project_slug}}/HELP.md

```diff
@@ -47,132 +47,7 @@ make jupyter
 ### Kill Jupyter
 
 ```shell 
-make kill-jupyter
+neuro-flow kill jupyter
 ```
 
 * The job with Jupyter Notebooks is terminated. The notebooks are saved on the platform storage. You may run `make download-notebooks` to download them to the local `notebooks/` directory.
-
-### Help
-
-```shell 
-make help
-```
-
-* The list of all available template commands is printed.
-
-## Data
-
-### Uploading to the Storage via Web UI
-
-On local machine, run `make filebrowser` and open the job's URL on your mobile device or desktop.
-Through a simple file explorer interface, you can upload test images and perform file operations.
-
-### Uploading to the Storage via CLI
-
-On local machine, run `make upload-data`. This command pushes local files stored in `./data`
```

Comment:
mb lets leave these hints too?
just replacing `make` to `neuro-flow upload data` to upload **data** directory to the neuro storage.

Reply:
ehhh I realized that this doc needs complete re-thinking, especially considering the fact that so far we didn't stabilize the template (there are still many things we want to drop/modify). So first, let's delete it.